awesomeConfig.cmake

```diff
@@ -332,6 +334,7 @@ if(DO_COVERAGE)
                     ${BUILD_DIR}/${file}
                     COPYONLY)
     endforeach()
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -g -O0 --coverage -fprofile-arcs -ftest-coverage")
```

Comment:
In that case this should perhaps be one commit changing the flags in `.travis.yml` and another that just moves (remaining) things to `CMakeLists.txt` without actually changing anything.